packages/search-spa/src/assets/icons/home.svg

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
```

Comment:
Can you remove the icon from the service? Add it to the global assets, and use absolute path of that for indexing if needed.

Reply:
Removed asset and updating the same is asset spa